_layouts/product-get-started-old.html.slim

```diff
@@ -46,9 +46,14 @@ br
           = page._extra_installation_instructions
     - else 
       li
-        asciidoc:
-          #{page.product.abbreviated_name} is only available through a full Red Hat subscription. Download #{page.product.abbreviated_name} by
-          https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[logging in to the Red Hat Customer Portal]  and downloading it from there.
+        - if page.product.id == 'brms' || page.product.id == 'bpmsuite'
```

Comment:
Isn't there a way of doing this without hard-coding these product codes into the template? Look at how other pages achieve this.